addons/io_hubs_addon/components/operators.py

```diff
@@ -634,38 +635,41 @@ def poll(cls, context):
                 return False
 
         return True
-
-    def draw(self, context):
-        layout = self.layout
+    
+    def draw(self, context): #this def is obsolete, bc. target_property is hidden by default now and relative_path is displayed by default anyways
```

Comment:
Is this is not required anymore in any of the supported Blender versions, I think we can safely remove it.